scss/_header.scss

```diff
@@ -0,0 +1,151 @@
+@import 'variable';
+
+.header {
+  height: 150px;
```

Comment:
height tầm 90px thôi, để 150px nó hơi to

Reply:
Em cũng thấy hơi to, nhưng mà trong figma để thế nên em cũng giữ nguyên mẫu